tests/unit/php/Core/Framework/Resources/config/test_entity_definition_compiled.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="true"/>
+
+        <service id="Shopware\Core\System\SalesChannel\Entity\SalesChannelDefinitionInstanceRegistry">
+            <argument type="string"/>
+            <argument type="string"/>
+            <argument type="string"/>
+            <argument type="string"/>
+        </service>
+
+        <service id="Shopware\Core\System\Country\CountryDefinition"
+                 class="Shopware\Core\System\Country\CountryDefinition">
```

Comment:
no need to specify the class when it's the same as the `id`, please remove all `class` tags in this file, that will make it shorter and easier to read

Reply:
yes, they are dependency, which are not resolved in the compiler class.